js/index.js

```diff
@@ -1,42 +1,135 @@
-    "product-h4":"Product",
-    "product-content": "Product content elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi. Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.",
-    "vision-h4":"Vision",
-    "vision-content": "Vision content elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi. Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.",
+    "services-h4": "Services",
+    "services-content":
+      "Services content elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi. Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.",
+    "product-h4": "Product",
+    "product-content":
+      "Product content elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi. Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.",
+    "vision-h4": "Vision",
+    "vision-content":
+      "Vision content elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi. Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis."
   },
-  "contact": {
-    "contact-h4" : "Contact",
-    "address" : "123 Way 456 Street Somewhere, USA",
-    "phone" : "1 (888) 888-8888",
-    "email" : "sales@greatidea.io",
-  },
-  "footer": {
-    "copyright" : "Copyright Great Idea! 2018"
+  contact: {
+    "contact-h4": "Contact",
+    address: "123 Way 456 Street Somewhere, USA",
+    phone: "1 (888) 888-8888",
+    email: "sales@greatidea.io"
   },
+  footer: {
+    copyright: "Copyright Great Idea! 2018"
+  }
 };
+//Selectors
 
 // Example: Update the img src for the logo
 let logo = document.getElementById("logo-img");
-logo.setAttribute('src', siteContent["nav"]["img-src"])
+logo.setAttribute("src", siteContent["nav"]["img-src"]);
+
+//Updating header & main content code snippet images
+let headerImg = document.getElementById("cta-img");
+headerImg.setAttribute("src", siteContent["cta"]["img-src"]);
+
+let middleImg = document.getElementById("middle-img");
+middleImg.setAttribute("src", siteContent["main-content"]["middle-img-src"]);
+
+//Header
+const nav = document.querySelectorAll("nav");
+const navItems = document.querySelectorAll("nav a");
+//adding 2 nav elements
+const newNavItem = document.createElement("a");
+const newNavItem2 = document.createElement("a");
+document.querySelector("header nav").appendChild(newNavItem).innerText =
+  siteContent["nav"]["nav-item-7"];
+document.querySelector("header nav").appendChild(newNavItem2).innerText =
```

Comment:
```suggestion
newNavItem2.innerText = siteContent["nav"]["nav-item-8"];
document.querySelector("header nav").prepend(newNavItem2);
```
So one of these needed to be prepended, added before the nav. However prepend has no return value, so the textContent property can't be chained. So add the textContent to the newItem first, then prepend it to the nav